marklogic-data-hub/src/main/java/com/marklogic/hub/legacy/collector/DiskQueue.java

```diff
@@ -162,15 +163,17 @@ private static void closeQuietly(Closeable obj) {
 
     private void openFile() throws IOException {
         if (fileQueue == null) {
-            fileQueue = File.createTempFile(DiskQueue.class.getSimpleName() + "-backingstore-", null, tempDir);
+            fileQueue = File.createTempFile(com.marklogic.hub.collector.DiskQueue.class.getSimpleName() + "-backingstore-", null, tempDir);
             fileQueue.deleteOnExit();
```

Comment:
Is there any reason the prefix of the temp file is changed to "com.marklogic.hub.collector.DiskQueue.class.getSimpleName()" rather the existing one "com.marklogic.hub.legacy.collector.DiskQueue.class.getSimpleName()"  for the legacy DiskQueue?

Reply:
Whoops. Incorrect auto-classpath rendering. While it makes sense for now, in the near future we won't be duplicating like we are now, fixed.